### DISCUSSION TITLE
Anyone else facing version issues with reaction-cli

### DISCUSSION THREAD

**Discussion by irtaza780** [reactioncommerce/reaction#6857] [Q&A]
So I have a lot of issues with reaction-cli.
First, reaction-cli installs using npm v14 or v15. Then to install the packages for reaction-backend API project, the required version is 18. To again, run the project, again version v14 or v15 is required since that is what's used to install reaction-cli. 

npm ERR! code ETARGET
npm ERR! notarget No matching version found for @reactioncommerce/api-core@3.0.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist

The following packages are not published on npmjs and cloning these as my own repos cause issues as well:
1. api-plugin-sequences
2. api-plugin-fulfillment
3. api-plugin-fulfillment-type-shipping
4. api-plugin-fulfillment-type-pickup
5. api-plugin-fulfillment-method-shipping-flat-rate
6. api-plugin-fulfillment-method-shipping-dynamic-rate
7. api-plugin-fulfillment-method-pickup-store
8. api-plugin-address-validation-test
9. api-plugin-bull-queue

**Comment by rjalili** [comment]
I have the same issue. Also, seems Node 18.10 is required which conflicts with the error saying that Node v14 or 15 is needed for setting up the project.

**Comment by mongy92** [comment]
@irtaza780 did you fix it?

**Comment by 1two3code** [comment]
I struggled with this myself for a few hours until I realized that the "quickstart" documentation _is not up to date_.
Instead of using the `reaction-cli` package, just pull the `reaction`, `example-storefront` and `reaction-admin` repositories from this github org: https://github.com/reactioncommerce

1. Install all the stuff
    - `pnpm i` in the `reaction` repo
    - `yarn` in the `example-storefront` repo
    - `npm i` in the `reaction-admin` repo
2. Start all the services (in dev mode)
    - `docker compose up -f docker-compose.dev.yml` in `reaction` root directory.
    - `npm run start:dev` in `reaction/apps/reaction` directory.
    - `yarn start:dev` in the `example-storefront` root directory.
    - `npm run start:dev` in the `reaction-admin` root directory.
    
Hope this helps.

**Reply by foobarbecue** [reply]
should be `docker compose -f docker-compose.dev.yml up`

**Comment by rjalili** [comment]
Thank you so much for this.

**Comment by foobarbecue** [comment]
I also found that I had to add `link-workspace-packages=true` to the `.npmrc` of the `reaction` repo .

I also had to set `export NODE_OPTIONS=--openssl-legacy-provider` for the example app.
